spec/system/polis_spec.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "pol.is", type: :system do
+  include_context "with a component"
+
+  let(:site_id) { ENV["POLIS_SITE_ID"] }
```

Comment:
Hi @Dynnammo, to me it is better to use static variables in tests to ensure a common behaviour

```suggestion
  let(:site_id) { "fake_site_id" }
```

Reply:
Agree. I'll check it by the end of the week.